pkg/machines/components/vnc.jsx

```diff
@@ -24,34 +24,35 @@ import { vmId } from '../helpers.es6';
 
 const _ = cockpit.gettext;
 
-const Frame = ({ url, containerId }) => {
+const Frame = ({ url, novncContainerId }) => {
     return (
-        <iframe src={url} className='machines-console-frame-vnc' frameBorder='0' name={containerId} data-container-id={containerId}>
+        <iframe src={url} className='machines-console-frame-vnc' frameBorder='0' name={novncContainerId} data-container-id={novncContainerId}>
             {_("Your browser does not support iframes.")}
         </iframe>
     );
 };
 Frame.propTypes = {
     url: React.PropTypes.string.isRequired,
-    containerId: React.PropTypes.string.isRequired,
+    novncContainerId: React.PropTypes.string.isRequired,
 };
 
 const Vnc = ({ vm, consoleDetail }) => {
     if (!consoleDetail) {
         return null;
     }
-    const uniqueFrameContainerId = `${vmId(vm.name)}-novnc-frame-container`;
+    const vmIdPrefix = vmId(vm.name);
+    const novncContainerId = `${vmIdPrefix}-novnc-frame-container`;
 
     const encrypt = window.location.protocol === "https:";
     const port = consoleDetail.tlsPort || consoleDetail.port;
 
-    let params = `?host=${consoleDetail.address}&port=${port}&encrypt=${encrypt}&true_color=1&resize=true&containerId=${encodeURIComponent(uniqueFrameContainerId)}`;
+    let params = `?host=${consoleDetail.address}&port=${port}&encrypt=${encrypt}&true_color=1&resize=true&containerId=${encodeURIComponent(vmIdPrefix)}`;
```

Comment:
In fact, it's no more element id but `prefix` to compose id for different elements within `vnc.html`.
I think we can rename the `containerId` URL param to `containerIdPrefix`.

Before this fix, access to just a single element (former containerId) was needed within vnc.html.
But to have scrolling work, I need to adjust multiple of them to achieve maximum visibility of the iframe _and_ related components (like VM name).

Please let me know, I can rename the URL param.